obspy/io/sac/tests/test_core.py

```diff
@@ -84,6 +84,12 @@ def test_read_xy_write_xy_via_obspy(self):
             tempfile = tf.name
             tr.write(tempfile, format='SACXY')
             tr1 = read(tempfile)[0]
+
+        sac = tr.stats.pop('sac', None)
+        sac1 = tr1.stats.pop('sac', None)
+        del sac
+        del sac1
```

Comment:
Nah, that's just what IPython does, print the entered commands return value's string representation (`repr(value)`, or `value._repr_pretty_()` if it's defined -- like we do for e.g. `Stream`).
If executed with python "normally" no return values ever get printed.